services/netlify/netlify.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const { renderBuildStatusBadge } = require('../build-status')
+const { BaseSvgScrapingService } = require('..')
+
+const pendingStatus = 'building'
+const notBuiltStatus = 'not built'
+
+const statusMap = {
+  building: pendingStatus,
+  waiting: pendingStatus,
+  initiated: pendingStatus,
+  stopped: notBuiltStatus,
+  ignored: notBuiltStatus,
+  blocked: notBuiltStatus,
+  infrastructure_failure: 'failed',
+}
+
+module.exports = class Netlify extends BaseSvgScrapingService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'netlify',
+      pattern: ':projectId',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Netlify',
+        namedParams: {
+          projectId: 'e6d5a4e0-dee1-4261-833e-2f47f509c68f',
```

Comment:
Are Netlify users well versed in how to retrieve their project id? If not, would it be helpful to include some inline docs/link to docs to give them guidance on how to retrieve it? 

Adding a documentation key like the example below would make the info visible to users directly in Shields (in the badge creation window for Netlify)
```suggestion
          projectId: 'e6d5a4e0-dee1-4261-833e-2f47f509c68f',
          documentation: 'To locate your project id, go to....',
```

Reply:
Makes sense, I'm adding that.